lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -667,8 +674,7 @@ struct FpToFpOpConversion
     };
     int inVecWidthBits = 32;
     int outVecWidthBits = 32;
-    if (srcTy.isFloat8E4M3FNUZ() ||
-        (computeCapability >= 90 && srcTy.isFloat8E5M2())) {
+    if (srcTy.isFloat8E4M3FNUZ()) {
```

Comment:
There is still f16 case that use 16bits inputs

Reply:
Should be fixed now